docs/content/guides/dagster/code-references.mdx

```diff
@@ -113,3 +113,124 @@ Each of the code references to `manual_references.py` will be visible in the **A
     height={604}
   />
 </center>
+
+---
+
+## Converting code references to link to a git repository
+
+In a local context, it is useful to specify local code references in order to navigate directly to the source code of an asset. However, in a production environment, you may want to link to the source control repository where the code is stored.
+
+### In Dagster Plus
+
+If using Dagster Plus, you can use the `link_code_references_to_git_if_cloud` utility to conditionally convert local file code references to source control links. This utility will automatically detect if your code is running in a Dagster Cloud environment and convert local file code references to source control links, pointing at the commit hash of the code running in the current deployment.
+
+```python
```

Comment:
This snippet should go in a .py file.

Reply:
Right now our docs build doesn't include cloud packages, so this won't work in a snippet file. Can file as a todo?

Reply:
Ah yeah - worth adding a comment to explain why we're doing this.